operators/samples/tenantRS.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: crownlabs.polito.it/v1alpha2
```

Comment:
I would drop this additional sample (also the one below), unless of any specific use.

Reply:
I would argue to keep this sample because it usefully tests specific tenant update/creation behavior that is isolated from other general namespaces (like in `john.doe` where `tea` and `coffee` need to be created/maintained). It is a more accurate minimal test case. However I would agree about deleting the second example

Reply:
Resolved in [ce41417](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/ce414173cb2405b5a4f0a5a540ce8cbc4f415463)